packages/hosted-form-v2/src/hosted-field.ts

```diff
@@ -180,7 +175,9 @@ export default class HostedField {
             .map((link) => link.href);
     }
 
-    private _isSubmitErrorEvent(event: any): event is HostedInputSubmitErrorEvent {
-        return event.type === HostedInputEventType.SubmitFailed;
+    private _isSubmitManualOrderErrorEvent(
+        event: any,
```

Comment:
```suggestion
        event: unknown,
```

Preferably avoid `any` as `unknown` forces you to perform more checks to make sure `event` is an object with `type` key. i.e.: 

```ts
if (typeof event !== 'object' || event !== null || 'type' in event === false) {
    return false;
}
```

Reply:
Changing this to `unknown` results in build error.
![Screenshot 2024-07-16 at 3 18 56 PM](https://github.com/user-attachments/assets/2cd2997d-547a-4de4-88a4-229e1cb86f06)

Reply:
Yes, you will need to add something like [this](https://github.com/bigcommerce/checkout-sdk-js/pull/2549#discussion_r1678699052) before the return statement to narrow down the type first.

Reply:
Added the checks, but kept any since there are build errors on using `unknown` here.